packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -189,168 +181,154 @@ const AddressDetails = ({
-                                                        <MobileWrapper>
-                                                            <SelectNative
-                                                                placeholder={localize('Please select')}
-                                                                label={localize('State/Province')}
-                                                                value={address_state_to_display || values.address_state}
-                                                                list_items={states_list}
-                                                                use_text={true}
-                                                                onChange={(e: { target: { value: string } }) => {
-                                                                    setFieldValue(
-                                                                        'address_state',
-                                                                        e.target.value,
-                                                                        true
-                                                                    );
-                                                                    setAddressStateToDisplay('');
-                                                                }}
-                                                                disabled={
-                                                                    disabled_items.includes('address_state') ||
-                                                                    (props.value?.address_state && has_real_account)
-                                                                }
-                                                            />
-                                                        </MobileWrapper>
-                                                    </>
-                                                )}
-                                            </Field>
-                                        ) : (
-                                            // Fallback to input field when states list is empty / unavailable for country
-                                            <InputField
-                                                name='address_state'
-                                                label={localize('State/Province')}
-                                                placeholder={localize('State/Province')}
+
+                                    <ThemedScrollbars height={height} className='details-form__scrollbar'>
+                                        <div className={classNames('details-form__elements', 'address-details-form ')}>
+                                            <FormInputField
+                                                name='address_line_1'
+                                                required={is_svg || is_mf}
```

Comment:
Does it mean, that it is just required for everyone?

Reply:
Will make this change. I think the condition can be removed